spec.bs

```diff
@@ -358,6 +387,8 @@ try {
 
   // response.data is a PublicKeyCredential, with a clientDataJSON that
   // contains the transaction data for verification by the issuing bank.
+  
+  Q: We need to define the format of Response.data somewhere in this doc, including what fields are present if the user accepts, of if they cancel/timeout.
```

Comment:
https://w3c.github.io/secure-payment-confirmation/#sctn-steps-to-respond-to-a-payment-request